lib/Runtime/Debug/TTEventLog.cpp

```diff
@@ -2541,10 +2545,23 @@ namespace TTD
         evt->Line = writeLocation.GetSourceLine();
         evt->Column = writeLocation.GetSourceColumn();
 
-        this->EmitLog(emitUri, emitUriLength);
+        this->EmitLog(emitUri, emitUriLength, entry);
+    }
+
+    bool EventLog::CanWriteInnerLoopTrace() const
```

Comment:
More pedantry than anything else but is the idea now that inner loop = record mode?